src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -135,8 +138,10 @@ protected virtual ParameterDescriptor CreateTriggerParameter(BindingMetadata tri
             }
             else
             {
-                throw new FunctionConfigurationException($"The binding type '{triggerMetadata.Type}' is not registered. " +
-                    $"Please ensure the type is correct and the binding extension is installed.");
+                string errorMessage = Host.ExtensionBundleManager.IsExtensionBundleConfigured()
```

Comment:
Rather than duplicate this logic, can you add a helper method CreateBindingError(string bindings) that works in both cases? It can use wording like above:

The binding type(s) '{bindings}' were not found in the configured extension bundle.
The binding type(s) '{bindings}' are not registered.